torch/csrc/distributed/autograd/utils.cpp

```diff
@@ -71,6 +74,18 @@ ContextPtr addRecvRpcBackward(
   return autogradContext;
 }
 
+Message getMessageWithProfiling(
+    const rpc::worker_id_t dstId,
+    torch::distributed::rpc::Message&& wrappedRpcMessage,
+    MessageType msgType,
+    const torch::autograd::profiler::ProfilerConfig& profilerConfig) {
+  auto wrappedProfilingMsg = std::make_unique<RpcWithProfilingReq>(
```

Comment:
What's the reason for creating a unique_ptr here instead of directly calling its ctor?

Reply:
Good point, I was looking at how it's done for RPC with autograd, but it's not clear we need a unique_ptr instead of direct stack allocation. Will update